src/main/java/com/google/cloud/tools/app/config/module/ListConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.tools.app.config.module;
+
+import com.google.cloud.tools.app.Option;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Configuration for {@link com.google.cloud.tools.app.module.ListAction}.
+ */
+public interface ListConfiguration {
+
+  /**
+   * Returns the modules collection to add to the command.
+   */
+  Collection<String> getModules();
+
+  /**
+   * Returns a map of Option, String pairs representing optional parameters.
+   */
+  Map<Option, String> getOptionalParameters();
```

Comment:
Actually, that is the case for List. Other commands have more parameters and could use with such a map. But here I could easily replace getOptionalParameters by getServer.

I'm wondering if having getOptionalParameters in some configs and not having them in others is weird. I guess not?